static/js/common.js

```diff
@@ -2,6 +2,23 @@ import $ from "jquery";
 
 import {i18n} from "./i18n";
 
+function toggle_password_visibility(password_field_id, password_selector) {
+    const password_field = $(password_field_id);
+
+    if (password_field.attr("type") === "password") {
+        password_field.attr("type", "text");
+    } else {
+        password_field.attr("type", "password");
+    }
+
+    $(password_selector).toggleClass("fa-eye fa-eye-slash");
```

Comment:
I think this would be better written as an if/else block rather than doing toggles.  Or probably even better, we should just make this two functions, one for `fa-eye` and the other for clicking on a `fa-eye-slash`.

As it is, the code is just kinda hard to read.